entrypoint.sh

```diff
@@ -377,10 +391,10 @@ function osd_disk_prepare {
   fi
 
   if [[ ! -z "${OSD_JOURNAL}" ]]; then
-    ceph-disk -v prepare ${OSD_DEVICE} ${OSD_JOURNAL}
+    ceph-disk -v prepare --cluster ${CLUSTER} ${OSD_DEVICE} ${OSD_JOURNAL}
```

Comment:
This is part of it. There is an open PR for the other part that will need to be brought in after its merged into ceph-docker.